cmd/edit/addon/cmd.go

```diff
@@ -206,9 +206,13 @@ func run(cmd *cobra.Command, argv []string) {
 			if addOnInstallationParam != nil {
 				dflt = addOnInstallationParam.Value()
 			}
-
+			question := param.Name()
+			// add a prompt to question name to indicate if the boolean param is required to be true
+			if param.ValueType() == "boolean" && param.Validation() == "^true$" {
+				question = fmt.Sprintf("%s (required to be true)", param.Name())
+			}
 			input := interactive.Input{
-				Question: param.Name(),
+				Question: question,
 				Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
 				Required: param.Required(),
```

Comment:
Can you ensure that `Required` is always `true` for these scenarios? Otherwise it will show `"Foo Param (required) (optional)"`

Reply:
thats not confusing at all :laughing: